src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1512,7 +1512,7 @@ FixedwingPositionControl::control_auto_takeoff(const hrt_abstime &now, const Vec
 		if (_param_fw_use_npfg.get()) {
 			_npfg.setAirspeedNom(target_airspeed * _eas2tas);
 			_npfg.setAirspeedMax(_param_fw_airspd_max.get() * _eas2tas);
-			_npfg.navigateWaypoints(prev_wp_local, curr_wp_local, curr_pos_local, ground_speed, _wind_vel);
+			_npfg.navigateWaypoints(curr_wp_local, curr_wp_local, curr_pos_local, ground_speed, _wind_vel);
```

Comment:
could you maybe add a small code comment here on why this change was necessary - like // pass curr_wp_local instead of prev_wp_local to disable line tracking and instead make it track the takeoff WP as a single point

Reply:
was already on it :) -- see recent force push